supertokens_python/recipe/thirdpartypasswordless/utils.py

```diff
@@ -11,21 +11,34 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
+
 from __future__ import annotations
 
 from typing import TYPE_CHECKING, Any, Awaitable, Callable, Dict, List, Union
 
```

Comment:
Missing tests for this PR

Reply:
Writing tests for `tppless`. Will push soon.

Reply:
Added. You may check `tests/thirdpartypasswordless/test_emaildelivery.py`